src/Bundle/CoreBundle/DependencyInjection/UniteCMSCoreExtension.php

```diff
@@ -31,5 +31,9 @@ public function load(array $configs, ContainerBuilder $container)
         // Set maximum nesting level as first argument to schema type manager and maximum nesting type object type.
         $container->getDefinition('unite.cms.graphql.schema_type_manager')->setArgument(0, $config['maximum_nesting_level']);
         $container->getDefinition('UniteCMS\CoreBundle\SchemaType\Types\MaximumNestingLevelType')->setArgument(0, $config['maximum_nesting_level']);
+
+        // Set maximum query limit as fifth argument to schema type manager and maximum nesting type object type.
+        $container->getDefinition('UniteCMS\CoreBundle\SchemaType\Types\QueryType')->setArgument(5, $config['maximum_query_limit']);
```

Comment:
When replacing a service argument here, there should be a default argument (e.g. 100) in the services.yaml definition of the service as a fallback.